### ISSUE TITLE
Error after upgrading to 2.4.4 around type annotations

### ISSUE THREAD

**Issue by trask** [immutables/immutables#574]
Hi, after upgrading from 2.4.3 to 2.4.4, I'm getting this error, looks like due to some code not expecting type annotations:

```
[ERROR] org.immutables.value.internal.$processor$.$Processor threw java.lang.IllegalArgumentException: Cannot parse type from input string '(@org.checkerframework.checker.tainting.qual.Untainted :: java.lang.String)'. unexpected term '('
[ERROR] at org.immutables.value.internal.$processor$.encode.$Type$Parser$1Reader.type($Type.java:751)
[ERROR] at org.immutables.value.internal.$processor$.encode.$Type$Parser.doParse($Type.java:852)
[ERROR] at org.immutables.value.internal.$processor$.encode.$Type$Parser.parse($Type.java:700)
[ERROR] at org.immutables.value.internal.$processor$.meta.$FromSupertypesModel.isEligibleFromType($FromSupertypesModel.java:136)
[ERROR] at org.immutables.value.internal.$processor$.meta.$FromSupertypesModel.<init>($FromSupertypesModel.java:82)
[ERROR] at org.immutables.value.internal.$processor$.meta.$ValueType.getBuildFromTypes($ValueType.java:1228)
[ERROR] at org.immutables.value.internal.$processor$.$Generator_Immutables._t18__generateBuilder($Generator_Immutables.java:4812)
[ERROR] at org.immutables.value.internal.$processor$.$Generator_Immutables$FragmentDispatch.run($Generator_Immutables.java:16115)
[ERROR] at org.immutables.value.internal.$generator$.$Templates$Fragment.invoke($Templates.java:248)
[ERROR] at org.immutables.value.internal.$generator$.$Intrinsics.$($Intrinsics.java:96)
[ERROR] at org.immutables.value.internal.$processor$.$Generator_Immutables._t4__generateImmutable($Generator_Immutables.java:579)
[ERROR] at org.immutables.value.internal.$processor$.$Generator_Immutables$FragmentDispatch.run($Generator_Immutables.java:16101)
[ERROR] at org.immutables.value.internal.$generator$.$Templates$Fragment.invoke($Templates.java:248)
[ERROR] at org.immutables.value.internal.$generator$.$Intrinsics.$($Intrinsics.java:96)
[ERROR] at org.immutables.value.internal.$processor$.$Generator_Immutables$1.run($Generator_Immutables.java:23)
[ERROR] at org.immutables.value.internal.$generator$.$Templates$Fragment.invoke($Templates.java:248)
[ERROR] at org.immutables.value.internal.$generator$.$Output$7.invoke($Output.java:163)
[ERROR] at org.immutables.value.internal.$generator$.$Intrinsics.$($Intrinsics.java:96)
[ERROR] at org.immutables.value.internal.$processor$.$Generator_Immutables._t0__generate($Generator_Immutables.java:20)
[ERROR] at org.immutables.value.internal.$processor$.$Generator_Immutables$FragmentDispatch.run($Generator_Immutables.java:16097)
[ERROR] at org.immutables.value.internal.$generator$.$Templates$Fragment.invoke($Templates.java:248)
[ERROR] at org.immutables.value.internal.$generator$.$AbstractGenerator.invoke($AbstractGenerator.java:57)
[ERROR] at org.immutables.value.internal.$processor$.$Processor.process($Processor.java:63)
[ERROR] at org.immutables.value.internal.$generator$.$AbstractGenerator.process($AbstractGenerator.java:87)
[ERROR] at org.immutables.processor.ProxyProcessor.process(ProxyProcessor.java:72)
```

**Comment by elucash** [comment]
the probable fix released in 2.4.6

**Comment by trask** [comment]
Yes, this is fixed now.  Thanks.
